packages/v4/src/content/design-guidelines/usage-and-behavior/filters/filters.md

```diff
@@ -12,210 +12,315 @@ related: [
 
-| Value selector | Trigger |
-| ------- | ------ |
-| Search field | The user presses the Enter key after typing in the input field. This action will create a filter for the first value shown in the filtered list. |
-| Select lists | The user selects or deselects an item in the list. |
+#### Supported features
+Faceted filters support [filter chips](#filter-chip), [badges](#badge-count), and [type-ahead](#type-ahead). 
 
+**Filter chips** are recommended as a way for users to see their selections when the menu is closed.
 
-### Filter chips
-When filters are applied, filter chips may be used to provide feedback.
+**Badges** can be used at the toggle level to indicate how many selections were made within the menu.
 
-**Basic filter chips**
+**Type-ahead** can be used as a way for users to narrow down the list of options when looking for the value they want to select.
 
-As filters are applied, the value will be displayed as a filter chip. Each filter chip has an ‘x’ to remove the filter, or users may click “Clear filters” to clear all applied filters at once.
+## Filter features
+Different attributes can be added to different filters. These include:
+* [Filter chips](#filter-chip)
+* [Badge count](#badge-count)
+* [Type-ahead](#type-ahead)
 
-<img src="./img/basic-filter-chips.png" alt="Basic filter chips" />
+This table describes which features are **supported** for each filter type. Note that just because a feature is supported, does not mean it must be used in your filters. 
 
-**Complex filter chips**
+|  | Filter chips | Type-ahead | Badge count
+| ----------------- | ------- | ------ | ------- | 
+| **Text entry**| Yes (if within an attribute filter) | Yes | No |
+| **Single select** | No | Yes | No |
+| **Checkbox select** | Yes | No (but you can have a search inside the menu) | Yes |
+|  **Attribute-value** | Yes | Yes (except for checkbox selects) | Yes (only for checkbox select)
+|  **Filter group** | Yes (except if all filters are single selects) | Yes (only for single selects) | Yes (only for checkbox selects) |
+|  **Toggle group** | No | No | No|
+|  **Faceted filter** | Yes | Yes | Yes|
 
-Filter chips may be used to show Boolean relationships between different attribute-value pairs. Individual filter chips can be removed by clicking the ‘x’ in each chip, and entire filter groups can be removed using the group ‘x’.
+### Filter chip
+When filters are applied, filter chips may be used to provide feedback to the user on what they are filtering on.
 
-<img src="./img/attribute-value-multi-chips.png" alt="Attribute value multi chips" />
+Filter chips may be used to show boolean relationships between different attribute-value pairs. Individual filter chips can be removed by clicking the ‘x’ in each chip, and entire filter groups can be removed using the group ‘x’.
 
-<img src="./img/faceted-combo-chips.png" alt="Faceted combo chips" />
+<img src="./img/chips.png" alt="chip groups" />
```

Comment:
I might help if this example were better aligned with the descriptions below. You mention how status and vendor combine, but the Vendor if not shown in the mockup.

Reply:
Oh good catch! Let me add that in